packages/account/src/providers/provider.ts

```diff
@@ -603,8 +606,37 @@ Supported fuel-core version: ${supportedVersion}.`
       return gqlClient.request(query, vars);
     };
 
+    const customOperations = (requester: Requester) => ({
+      getBlobs(variables: { blobIds: string[] }) {
+        const queryParams = variables.blobIds.map((_, i) => `$blobId${i}: BlobId!`).join(', ');
+        const blobParams = variables.blobIds
+          .map(
+            (_, i) => `blob${i}: blob(id: $blobId${i}) {
+                        id
+                      }`
+          )
```

Comment:
Nit - maybe we don't need to worry about line breaks here.

```suggestion
          .map((_, i) => `blob${i}: blob(id: $blobId${i}) { id }`)
```